content/release-10-11-0/streaming-analytics-10-11-0-bundle/10_11_0_5.md

```diff
@@ -0,0 +1,33 @@
+---
+weight: 35
+title: Release 10.11.0.5
+layout: redirect
+---
+
+This release upgrades the Apama-ctrl microservice to use Apama 10.11.2.0 (which is the same as Apama 10.11.0 Fix 5).
+
+### Fixes
+
+<table>
+<colgroup>
+    <col style="width: 15%;">
+    <col style="width: 70%;">
+    <col style="width: 15%;">
+</colgroup>
+<thead>
+<tr>
+<th style="text-align:left">Component</th>
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td style="text-align:left">Apama-ctrl microservice</td>
+<td style="text-align:left">Spring Framework has been upgraded to 5.3.18 and Spring Boot has been upgraded to 2.5.12 to resolve security vulnerabilities CVE-2022-22965.</td>
```

Comment:
And only mention Spring Framework

This version doesn't upgrade spring boot.

Reply:
Ok, removed this. But the readme text on https://itrac.eur.ad.sag/browse/PAB-3070 still mentions Spring Boot ...???